Sources/PointFree/Rss.swift

```diff
@@ -0,0 +1,273 @@
+import Foundation
+import Html
+import Optics
+import Prelude
+
+struct RssChannel {
+  var copyright: String
+  var description: String
+  var image: Image
+  var itunes: Itunes?
+  var language: String
+  var link: String
+  var title: String
+
+  struct Image {
+    var link: String
+    var title: String
+    var url: String
+  }
+
+  struct Itunes {
+    var author: String
+    var categories: [Category]
+    var explicit: Bool
+    var keywords: [String]
+    var image: Image
+    var owner: Owner
+    var subtitle: String
+    var summary: String
+    var type: ChannelType
+
+    enum ChannelType: String {
+      case episodic
+      case serial
+    }
+
+    struct Image {
+      var href: String
+    }
+
+    struct Category {
+      var name: String
+      var subcategory: String
+    }
+
+    struct Owner {
+      var email: String
+      var name: String
+    }
+  }
+}
+
+struct RssItem {
+  var description: String
+  var dublinCore: DublinCore?
+  var enclosure: Enclosure?
+  var guid: String
+  var itunes: Itunes?
+  var link: String
+  var media: Media?
+  var pubDate: Date
+  var title: String
+
+  struct DublinCore {
+    var creators: [String]
+  }
+
+  struct Enclosure {
+    var length: Int
+    var type: String
+    var url: String
+  }
+
+  struct Itunes {
+    var author: String
+    var duration: Int
+    var episode: Int
+    var episodeType: EpisodeType
+    var explicit: Bool
+    var image: String
+    var subtitle: String
+    var summary: String
+    var season: Int
+    var title: String
+
+    enum EpisodeType: String {
+      case bonus
+      case full
+      case trailer
+    }
+  }
+
+  struct Media {
+    var content: Content
+    var title: String
+
+    struct Content {
+      var length: Int
+      var medium: String
+      var type: String
+      var url: String
+    }
+  }
+}
+
+func node(rssChannel channel: RssChannel, items: [RssItem]) -> Node {
+  let itunesNodes = channel.itunes.map(nodes(itunes:)) ?? []
+
+  return node(
+    "channel",
+    [
+      node("title", [text(channel.title)]),
```

Comment:
right now I'm mostly just using the deep embedding of `Node` into HTML to generate the RSS, but I imagine a follow up PR could clean this up by providing a shallow embedded with RSS-specific functions.